src/Background.cpp

```diff
@@ -85,6 +87,7 @@ namespace Background {
 	{
 	}
 
+	static size_t s_texture0Name = Graphics::Renderer::GetName("texture0");
```

Comment:
This was intended to be a C++17 `constexpr` string hashing function inline instead of storing name hashes as static variables; I started writing this under C++11 and never found a good hash library to integrate. Work for the future.